src/main/java/core/basesyntax/web/filter/AuthenticationFilter.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.web.filter;
+
+import core.basesyntax.lib.Injector;
+import core.basesyntax.service.DriverService;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AuthenticationFilter implements Filter {
+    private static final String DRIVER_ID = "driver_id";
+    private static final Injector injector = Injector.getInstance("core.basesyntax");
+    private final Set<String> allowedUrls = new HashSet<>();
+    private final DriverService driverService = (DriverService) injector
```

Comment:
we don't use it as well as injector remove both of these fields

Reply:
done